### ISSUE TITLE
Tests take forever to run

### ISSUE THREAD

**Issue by tseaver** [buildout/buildout#230]
[14 - 23 minutes for each environment on Travis](https://travis-ci.org/buildout/buildout/builds/44200649)

**Comment by mgedmin** [comment]
Ideas for speeding things up:

- add `sudo: false` to .travis.yml (see http://docs.travis-ci.com/user/workers/container-based-infrastructure/)
- cache CPython builds (see http://docs.travis-ci.com/user/caching/#Arbitrary-directories)

**Comment by reinout** [comment]
I'm trying out the travis caching in a branch now. Once I got it working I'll submit a pull request. That should shave 2-3 minutes off the travis build time. Running the actual tests still takes the most time, though.

**Comment by reinout** [comment]
Yeah! My improvements work. An old run (https://travis-ci.org/buildout/buildout/builds/44028728 for instance) might take up to 1 hour 20 minutes in total (for all five builds) to run.

Now (https://travis-ci.org/buildout/buildout/builds/69056029) it is at half an hour. Almost three times quicker.

If we want it even quicker, we need to look at the tests themselves. More offline work. Less hitting pypi (if that happens at all). Etc.

OK to close this issue for now?
